index.html

```diff
@@ -472,6 +249,61 @@ <h3>Contact me</h3>
             </div>
         </section>
     </div> 
+    <div class="container-modal">
+        <div class="single-modal">
+            <div class="top-modal">
+                <div class="modal-title">
+                    <h3>Tonic</h3>
+                    <img id="close-modal" src="./images/close-btn.png" style="height: 12px; width: 12px;cursor: pointer;"  alt="">
+                </div>
+                <ul>
+                    <li><a href="#">Canopy</a></li>
+                    <li class="tag-dot"><a href="#">Back end dev</a></li>
+                    <li class="tag-dot"><a href="#">2015</a></li>
+                </ul>
+                <img class="moda-image-all" src="images/modalpop.png"  alt="">
+            </div>
+            <div class="bottom-modal">
+                <div class="bottom-modal-desc">
+                    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has 
+                        been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the 
+                        leap into electronic typesetting, remaining essent</p>
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                
+                <div class="modal-btn-view">
+                    <a class="modal-mobile-btn see-live">See live <img src="./images/modal-live.png" alt="" style="height: 18px; width: 18px;"> </a>
+                    <a class="modal-mobile-btn see-source"> See source<img src="./images/modal-source.png" alt="" style="height: 18px; width: 18px;"></a>
+                </div>
+            </div>
```

Comment:
- As suggested by the previous reviewer and part of the project's requirement, you need to implement both your modal and project cards in javaScript. No content of the project card or modal should be in your html file. 🌸 
- You can move this entire modal code to your script.js file